app/models/planning_application.rb

```diff
@@ -292,6 +292,10 @@ def overdue_requests
     validation_requests.select { |req| req.overdue? && req.state == "open" }
   end
 
+  def invalidation_response_due
+    15.business_days.after(invalidated_at.to_date)
+  end
```

Comment:
More of a product question, but we already have methods for figuring out when each validation request is due. Would this not be the latest due date of any validation requests?

Reply:
It's a legal thing - we need to give the applicant 15 days from when they get the notification. So if a planner raised one or two validation requests then didn't do anything for a couple of days because they were checking something and then  invalidated the application, the date on the email would be wrong.

Reply:
Ah! Well in that case, then the due date for the validation requests should be the date they were sent then?

Reply:
Very good point! I'll add some extra logic for that.

Reply:
I wasn't thinking this through properly. Any validation requests that are bundled together in the invalidation notice will have the same due date as 15 days after the planning application `invalidated_at` date. So I could either add some new logic to look at all the validation requests and select the latest due date or I could simply call the planning_application method as above. Thoughts?

Reply:
Ah right, yes, we're sort of going in circles here of logic. We could probably tidy it up a bit for clarity, but this is at least correct.